src/integrations/admin/workouts-integration.php

```diff
@@ -98,6 +99,16 @@ public function register_hooks() {
 	 * @return array the filtered submenu pages.
 	 */
 	public function add_submenu_page( $submenu_pages ) {
+		// If Premium has an outdated version, which also adds a 'workouts' submenu, don't show the Premium submenu.
+		if ( $this->should_update_premium() ) {
+			$submenu_pages = array_filter(
+				$submenu_pages,
+				function ( $item ) {
+					return $item[4] !== 'wpseo_workouts';
+				}
+			);
+		}
+
```

Comment:
This way, with Premium active, the `Workouts` menu item is placed _after_ the `Redirects` menu item. Not a big deal but I'd try to solve it or at least ask Product if we really want to fix it.

We might just bump the priority of `Redirects` in the new Premium, so the order is OK when the user has everything up-to-date.

Another, related, thing: probably we should remove the "premium" badge fromt he `Workouts` menu item, though it makes more sense to do it late when the conf. workout will be added.

Reply:
I will discuss this with @Dieterrr and apply what is decided.

Reply:
> We might just bump the priority of Redirects in the new Premium, so the order is OK when the user has everything up-to-date.

I pushed a [commit](https://github.com/Yoast/wordpress-seo-premium/commit/a4bce26a24a60c661097323f662caee0876cd261) in Premium to fix this. It was decided with @Dieterrr that it is not worth the time to fix it for the mismatched version scenario.

> Another, related, thing: probably we should remove the "premium" badge fromt he Workouts menu item, though it makes more sense to do it late when the conf. workout will be added.

@Dieterrr will discuss whether we want to remove the premium badge from the menu item. He will let us know when he has an answer. This can indeed be picked up in a later issue.